### DISCUSSION TITLE
keyboard doesn't work !

### DISCUSSION THREAD

**Discussion by lnxfsf** [rustdesk/rustdesk#8668] [Q&A] [answered]
I'm on windows 10 (laptop Dell), I've installed latest version from site RustDesk. And on ubuntu 22.04 LTS there's also latest RustDesk (ASUS ExpertCenter PN52). Ubuntu is installed on device itself... (keyboard directly works.. on Ubuntu device )

But me, from Windows cant type anything on there.. I can use mouse. Mouse/keyboard is enabled... 

When trying TeamViewer, with TeamViewer keyboard works. So not sure why this doesn't work.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/wiki/FAQ#windows-uac

https://github.com/rustdesk/rustdesk/wiki/FAQ#keyboard-translation-modes

**Reply by lnxfsf** [reply]
by adding -qs- to name, it worked
